dotcom-rendering/src/web/server/frontTemplate.ts

```diff
@@ -154,13 +154,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 
                 <script type="module">
                     window.guardian.mustardCut = true;
-                    window.guardian.gaPath = "${gaPath.modern}";
-                </script>
-
-                <script nomodule>
-                    // Browser fails mustard check
-                    window.guardian.mustardCut = false;
```

Comment:
As per @sndrs 's comment above, we should keep this negative value to maintain the contract

Reply:
It’s back in!